docs/routes.md

```diff
@@ -202,6 +202,17 @@ snap name, if any.
       }
     }
 
+### Editing files
+
+These routes redirect to the appropriate GitHub URL to create or edit a file
+in the default branch of the repository:
+
+    GET /api/github/repos/:owner/:name/new?filename=:filename&value=:value
+    Cookie: <session cookie>
+
+    GET /api/github/repos/:owner/:name/edit?filename=:filename
```

Comment:
I find it a bit confusing to have API routes that are actually redirects to GitHub UI.

At this point we only need default branch name for editing/adding `snapcraft.yaml`. Maybe we could request this data just once (when we load snapcraft.yaml data for the snap/repo) and expose the branch name in `snapcraftData` in snap object?

This would make the default branch name available to client app, so the new/edit URL for snapcraft.yaml could be built without the need to redirect.